README.md

```diff
@@ -0,0 +1,46 @@
+# Redirect Action Example
+
+[![Quality](https://curity.io/assets/images/badges/redirect-action-example-quality.svg)](https://curity.io/resources/code-examples/status/) 
+[![Availability](https://curity.io/assets/images/badges/redirect-action-example-availability.svg)](https://curity.io/resources/code-examples/status/)
+
+This repository is an example of an Authentication Action Plugin for the
+Curity Identity Server, which uses the `redirect()` response type. It consists
+of two parts: 
+
+- the plugin code is in the `identityserver.plugins.authenticationactions.redirect-action`
+  directory
+- `example` directory which contains a simple demo website to which the
+  plugin action redirects. The demo is written in Java and Spring. 
+
+## Running the example
+
+To run the example follow these steps:
+
+- Start the project by running the command `./gradlew bootRun` from the `example`
+  directory.
+- Install the authentication plugin. Refer to the [plugin's README](identityserver.plugins.authenticationactions.redirect-action/README.rst)
+  to check how to do it.
+- When the plugin is installed go to the admin UI, then **Authentication Profile** -> **Authenticators**.
+  Edit an authenticator of your choice.
+- If you want to run the example with a local instance of the Spring application you'll need an http client which uses
+  the `http` scheme. To add such client click the `Facilities` button in the top right corner of the screen, scroll down
+  to **Http clients** and click the `New` button.
```

Comment:
here you do it too, so perhaps I'm missing some nuance of the formatting. If so, ignore the above comment.